src/utils/index.ts

```diff
@@ -1,6 +1,6 @@
 const createMajorVersionNumber = (num: string): string => {
   if (!num) {
-    throw new Error('Invalid format number given, it must match \'x.x.x\' format.');
+    throw new Error("Invalid format number given, it must match 'x.x.x' format.");
```

Comment:
Is sonar tube fine with the double quotes here?

Reply:
I'll check....

Reply:
SonarQube has no issues with it.